src/runtime/env.rs

```diff
@@ -368,3 +374,39 @@ fn migrate_storage_schema_to_v5<E: Env>() -> TryEnvFuture<()> {
         .and_then(|_| E::set_storage(SCHEMA_VERSION_STORAGE_KEY, Some(&5)))
         .boxed_env()
 }
+
+fn migrate_storage_schema_to_v6<E: Env>() -> TryEnvFuture<()> {
+    E::get_storage::<serde_json::Value>(PROFILE_STORAGE_KEY)
+        .and_then(|mut profile| {
+            match profile
+                .as_mut()
+                .and_then(|profile| profile.as_object_mut())
+                .and_then(|profile| profile.get_mut("settings"))
+                .and_then(|settings| settings.as_object_mut())
+                .map(|settings| (settings.remove("playInExternalPlayer"), settings))
+            {
+                Some((Some(play_in_external_player), settings)) => {
+                    settings.insert(
+                        "playerType".to_owned(),
+                        if play_in_external_player == "true" {
```

Comment:
I'm writing tests about the migration but I wanted to point out that this is incorrect just in case someone wants to merge the PR before I finish with the tests.

A `Value::Bool` cannot be compared with a str of `"true"`